docs/developer-guide/creating-loaders-and-writers.md

```diff
@@ -65,3 +65,17 @@ export async function parse3DTile(arrayBuffer, options, context) {
 Remarks:
 
 - While a loader could potentially import `parse` from `@loaders.gl/core` to invoke a sub-loader, it is discouraged, not only from a dependency management reasons, but it prevents loaders.gl from properly handling parameters and allow worker-loaders to call other loaders.
+
+## Accessing the Response object
```

Comment:
@ibgreen by serializing the response I can pass it to the worker in the same manner as the worker options. If you're OK with the approach, I'll add some tests.

https://github.com/visgl/loaders.gl/blob/fb92a14cbdc77f95e0af576010fcff54abbd5c0c/modules/core/src/lib/api/parse.ts#L67-L70

This data isn't too much overhead, but then again it isn't nothing and will be present on every load whether requested or not. Do you think it is worth enabling this behavior via a loader option?

Reply:
I've gone ahead and added some tests for the web worker case. In order to make it work I extended the `NullLoader` to also pass back the `options` and `context`. From what I can see in the code, it is only used for testing and this seems like the cleanest way to test using a worker.

Reply:
Well it was intended to be a public loader, mainly to make the API more orthogonal and give an "escape hatch". So far I haven't seen a big value for it though, and also I don't see it listed in the TOC. 

It is fine but if you are changing semantics perhaps it would be nice make that an opt-in on the NullLoader.

Reply:
I've made it opt-in